arbitrum/multigas/resources.go

```diff
@@ -24,128 +26,132 @@ type MultiGas struct {
 	refund uint64
 }
 
+// MultiGasPair represents a single resource kind and its associated gas amount.
+type MultiGasPair struct {
```

Comment:
I would just call this Pair or Entry

The package is already called multigas.

That way, callers will write multigas.Pair{} instead of multigas.MultiGasPair{}